src/flows/repo.rs

```diff
@@ -2,25 +2,28 @@ use crate::filesystem;
 use crate::fzf;
 use crate::git;
 use crate::structures::fzf::{Opts as FzfOpts, SuggestionType};
+use anyhow::Context;
+use anyhow::Error;
 use git2::Repository;
-use std::error::Error;
 use std::fs;
 use std::io::Write;
 use walkdir::WalkDir;
 
-pub fn browse() -> Result<(), Box<dyn Error>> {
-    let repo_path_str = format!("{}/featured", filesystem::tmp_path_str());
+pub fn browse() -> Result<(), Error> {
+    let repo_path_str = format!("{}/featured", filesystem::tmp_path_str()?);
 
-    filesystem::remove_dir(&repo_path_str);
-    filesystem::create_dir(&repo_path_str);
+    // The dir might not exist which would throw an error. But here we don't care about that
+    // since our purpose is to make space for a new directory. Potential other errors
+    // (e.g due to lack of permission) will cause an error during dir creation.
+    let _ = filesystem::remove_dir(&repo_path_str);
```

Comment:
Why using `let _ =`?

Reply:
Rust requires you to use the return value of `remove_dir`. But as stated in the comment above it, we don't care about the potential `Err` value. And the `Ok` type is `()` anyway.